src/swagger/parameters/path/languages.yml

```diff
@@ -0,0 +1,25 @@
+name: index
+in: path
+required: true
+description: |
+  The `index` of the language to get.
+schema:
+  type: string
+  enum:
```

Comment:
I like how  you enum the shorter lists.

Reply:
thanks! i figured anything with less than 30 options might as well put it in an enum